shared/command/cmd.go

```
@@ -133,6 +143,22 @@ func (r *Command) String() (s string) {
 	return
 }
 
+// options returns options with referenced params ${k} expanded.
+func (r *Command) options() (expanded []string) {
+	expanded = make([]string, len(r.Options))
+	for i, opt := range r.Options {
+		for k, v := range r.Param {
+			opt = strings.Replace(
```

golangci-lint (2.12.2)

[error] QF1004: could use strings.ReplaceAll instead

(staticcheck)